lib/active_force/errors.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module ActiveForce
+  Error = Class.new(StandardError)
```

Comment:
I think it would be a good idea to start subclassing all module-specific errors under this namespaced `ActiveForce::Error` instead of `StandardError`.  It gives the consumer the option to use more specific rescue conditions.